docs/nipoppy/workflow/dicom_org.md

```diff
@@ -32,7 +32,7 @@ This is a dataset specific process and needs to be customized based on local sca
     | 002            | ses-03  | MyStudy_002_2024      | sub-002  | sub-002 | False      | False     | False     |
 
 2. Download DICOM dumps (e.g. ZIPs / tarballs) in the `<DATASET_ROOT>/downloads` directory. Different visits (i.e. sessions) must be downloaded in separate sub-directories and ideally named as listed in the `global_config.json`. The DICOM download and extraction process is highly dataset-dependent, and we recommend using custom scripts to automate it as much as possible.
```

Comment:
I am not sure we standardized this at any point but currently we have `global_configs.json` (with an "s")

Reply:
Right, I'll change that to be plural. The name of the config file should be fixed I think?